db/migrate/20170203141631_add_maintenance_to_storages.rb

```diff
@@ -0,0 +1,5 @@
+class AddMaintenanceToStorages < ActiveRecord::Migration[5.0]
+  def change
+    add_column :storages, :maintenance, :boolean
```

Comment:
Gah it looks like there is still an issue if you have the datastore on two VCs and in a storage cluster in one of them, you can put that one into maintenance mode but the other will not be.